### DISCUSSION TITLE
Date Range on a Calculated Date&Time Column

### DISCUSSION THREAD

**Discussion by Gaztab** [microsoft-search/pnp-modern-search#2050] [Q&A]
**Is your feature request related to a problem? Please describe.**
I'm using PnP modern Search to filter documents on a document library within SharePoint Online. I want to filter the documents by a calculated column set up as Date & Time called "Expiry Date". The calculated column (Expiry Date) uses a date column, called Created Date and adds 365 days, then I add conditional formatting to show if the document is expired (hopefully I've explained this well). 
So, I've added the Expiry Date column to a refiner (Refinable01). I've then added this to my filters, and set it as "Date Range". However when I select any range it returns 0 results. No matter the range. I set the filter as a "Check Box" and it does display the date information but it displays it as: "datetime;#2022-07-23T23:00:00Z". 

**Describe the solution you'd like**
Is there a way to get the Date Range filter to work with the Calculated Column ?

**Describe alternatives you've considered**
is there a way to change how the "Check Box" filter displays the date? So take off the datetime;# and the time at the end, just leaving the date?

**Additional context**
Can provide if required. 

Thank you

**Comment by wobba** [comment]
From this old post https://social.technet.microsoft.com/Forums/en-US/145db16f-a547-4ec4-bd03-b80d77ebe886/how-to-map-calculated-column-to-managed-property?forum=sharepointsearch you see it won't work as the crawled property is not pure date string so mapping it to RefinableDate01 seems not to work.
